### PR TITLE
Include DAOS-4419 change in soak

### PR DESCRIPTION
**Pull request by mjean308**
Quick-Functional: true
Skip-func-hw-test-small: true
Skip-func-hw-test-medium: true
Skip-func-test-vm: true
Test-tag-hw-large: soak_smoke

Signed-off-by: Maureen Jean <maureen.jean@intel.com>

### CODE REVIEW

File: src/tests/ftest/soak/soak_utils.py [hunk 1]
```diff
@@ -588,9 +590,9 @@ def start_dfuse(
     dfuse_env = "export D_LOG_MASK=ERR;export D_LOG_FILE={}".format(dfuse_log)
     dfuse_start_cmds = [
         "mkdir -p {}".format(dfuse.mount_dir.value),
-        "clush -w $SLURM_JOB_NODELIST \"cd {};{};{}\"".format(
+        "clush -S -w $SLURM_JOB_NODELIST \"cd {};{};{}\"".format(
             dfuse.mount_dir.value, dfuse_env, dfuse.__str__()),
-        "sleep 10",
+        "sleep 20",
```
Comment (phender) [inline src/tests/ftest/soak/soak_utils.py:595]: Was this intended?

**Review by phender — CHANGES_REQUESTED** [review]

**Review by shimizukko — DISMISSED** [review]

**Review by dinghwah — DISMISSED** [review]
LGTM

**Review by shimizukko — APPROVED** [review]

**Review by dinghwah — APPROVED** [review]
LGTM.

**Comment by mjean308** [comment]
Single test failure is a soak failure due to known fio dfuse issue - DAOS-7778

**Comment by johannlombardi** [comment]
@sylviachanoiyee @mjean308 please note that the commit message of this PR is not compliant to our standard.

**Pull request merged**